.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This invalidates the [terra-eslint-config setup in our package.json](https://github.com/cerner/terra-framework/pull/573/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R27). Lets move this to the data-picker package and leverage this configuration for just that package.

More info can be found here: https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy

Reply:
Gotcha. I think what would benefit here is a .gitignore file for this package.